pkg/operator/k8sutil/k8sutil.go

```diff
@@ -70,3 +71,41 @@ func GetK8SVersion(clientset kubernetes.Interface) (*version.Version, error) {
 	}
 	return version.MustParseSemantic(serverVersion.GitVersion), nil
 }
+
+// MergeResourceRequirements merges two resource requirements together (first overrides second values)
```

Comment:
Should this function better placed in `k8sutil/resource.go`? Just to keep all resource requierement logic in one file.

Reply:
I moved the function and test to files named `k8sutil/resources....go`